src/modules/authentication/components/ChangePassword/CSS/ChangePassword.module.css

```diff
@@ -0,0 +1,43 @@
+
+.form {
+    margin-top: 10px;
+    border-radius: 16px;
+    background-color: #315951E5;
+}
+
+.form h2 {
+    color: #EF9B28;
+}
+input {
+  background-color: transparent;
+  margin-block: 20px;
+  border: none;
+  border-bottom: 1px solid #ffffff;
+  padding-block: 10px;
+  outline: none;
+  color: #ffffff;
+  width: 100%;
+}
+input::placeholder {
+  color: #ffffff;
+}
+.input-container {
+    position: relative;
+}
+.input-container i {
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    color: #8391A1;
+    cursor: pointer;
+    right: 10px; 
+}
+.custom-btn {
+    background-color: #EF9B28;
+    border-radius: 6rem;
+    border: none;
+    width: 100%;
+    height: 3.5rem;
+    font-size: 16px;
```

Comment:
For font sizes, it is recommended to use rem for better accessibility. Pixel size fonts don't scale when the user increase the font size of the browser.